src/app/public/modules/modal/modal.module.ts

```diff
@@ -50,13 +63,15 @@ import {
     SkyModalHostComponent
   ],
   providers: [
+    SkyDynamicComponentService,
```

Comment:
Why do we need to provide `SkyDynamicComponentService` and also import its module? Does the module not provide this service?

Reply:
We don't. I had a bug at one point that was self induced and I forgot to remove the provider. Fixed